### ISSUE TITLE
CC0 license in README

### ISSUE THREAD

**Issue by Blu3wolf** [OrionFive/Hospitality#1]
So, the CC0 license is an all or nothing thing. It by definition applies to you, not some third party - and it makes the statement that you are waiving all rights to your work. 

Creative Commons recommend that their licenses not be used for software, and for their own software, they use free and open source licenses, such as the Free Software Foundation's GNU General Public License. 

Its worth noting that as the sole creator of a work, nothing prevents you from releasing your work under multiple licenses. You are capable of releasing your work under a FOS license, while also granting a non exclusive license to Ludeon Studios to use your work in a commercial offering, separately from the FOS license. 

I would recommend licensing any art assets that are in the mod, under the CC license of your choice, and licensing the code assets separately, ideally under a FOSS license, such as the LGPL.

In short though, you can't really license your work as CC0 for one party,  CC-BY-SA 4.0I for everyone else. 

The only other quirky part is when you are not the sole author. For FOSS licensing, this is not really considered an issue, but if you want to retain sole rights to your work, I would recommend checking our a Contributor Licensing Agreement, that essentially requires any contributions to your mod (should you accept them) be offered under whichever license you are planning on using.

**Comment by OrionFive** [comment]
Thanks for chiming in! 

Reading these kinds of documents gives me headaches, so I just went with the same license some other mod used (is that already a licence violation?), that sounded roughly like what I wanted.

You sound like you know what you're talking about. Would you mind setting up the readme with the license you suggested?

**Comment by Blu3wolf** [comment]
I can make a pull request that would make proposed changes to the readme, sure. You could then examine those changes, and if you approve of them, merge the pull request into a branch. 

Ill go ahead and do that shortly.

**Comment by Blu3wolf** [comment]
Opened #2 with some explanation there. If you are happy with the changes, you can merge it into your project. 

To make your life easier, I am dropping all claims to the copyright on the text I wrote for the readme - and I did not write the text for the license. So that avoids any concerns there.

**Comment by OrionFive** [comment]
Why is copyright always such a nightmare... It's merged.
